spring-cloud-context/src/main/java/org/springframework/cloud/context/refresh/ConfigDataContextRefresher.java

```diff
@@ -92,9 +92,6 @@ protected void updateEnvironment() {
 			postProcessor.postProcessEnvironment(environment, application);
 		}
 
-		if (environment.getPropertySources().contains(REFRESH_ARGS_PROPERTY_SOURCE)) {
```

Comment:
Why is this removed?

Reply:
Because this `PropertySource` is added by `LegacyContextRefresher`, It's not involved if `ConfigDataContextRefresher` is using.